### ISSUE TITLE
markupsafe error

### ISSUE THREAD

**Issue by timandlife** [shawly/docker-nut#8]
Changing the markupsafe version to 2.0.1 fixes the issue

pip install markupsafe==2.0.1

**Comment by shawly** [comment]
What issue?

**Comment by timandlife** [comment]
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/nut/venv/lib/python3.10/site-packages/markupsafe/__init__.py)

**Comment by StrangerJona** [comment]
I have attempted the above, with a `sudo python3 -m pip install markupsafe==2.0.1`

and still getting the below

```
	Updating titledb...
-------------------------------------
Updating titledb, this might take some time...
Updating files: 100% (69/69), done.
HEAD is now at 2754f98 meta
Already up to date.
Importing titledb...
loading blacklist /nut/conf/blacklist.online.txt_
loading blacklist /nut/conf/blacklist.online.txt
Traceback (most recent call last):
  File "/nut/nut.py", line 18, in <module>
    from nut import Title
  File "/nut/nut/__init__.py", line 24, in <module>
    import Fs
  File "/nut/Fs/__init__.py", line 3, in <module>
    from Fs.Nsp import Nsp
  File "/nut/Fs/Nsp.py", line 22, in <module>
    from nut import blockchain
  File "/nut/nut/blockchain.py", line 25, in <module>
    from flask import Flask, jsonify, request
  File "/nut/venv/lib/python3.10/site-packages/flask/__init__.py", line 14, in <module>
    from jinja2 import escape
  File "/nut/venv/lib/python3.10/site-packages/jinja2/__init__.py", line 12, in <module>
    from .environment import Environment
  File "/nut/venv/lib/python3.10/site-packages/jinja2/environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "/nut/venv/lib/python3.10/site-packages/jinja2/defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "/nut/venv/lib/python3.10/site-packages/jinja2/filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/nut/venv/lib/python3.10/site-packages/markupsafe/__init__.py)

Done.
```

**Comment by StrangerJona** [comment]
What am i doing wrong here?
EDIT: I have installed markupsafe==2.0.1 too, and still getting below.

```
Traceback (most recent call last):
  File "/nut/nut.py", line 18, in <module>
    from nut import Title
  File "/nut/nut/__init__.py", line 24, in <module>
    import Fs
  File "/nut/Fs/__init__.py", line 3, in <module>
    from Fs.Nsp import Nsp
  File "/nut/Fs/Nsp.py", line 22, in <module>
    from nut import blockchain
  File "/nut/nut/blockchain.py", line 25, in <module>
loading blacklist /nut/conf/blacklist.online.txt
    from flask import Flask, jsonify, request
  File "/nut/venv/lib/python3.10/site-packages/flask/__init__.py", line 14, in <module>
    from jinja2 import escape
  File "/nut/venv/lib/python3.10/site-packages/jinja2/__init__.py", line 12, in <module>
    from .environment import Environment
  File "/nut/venv/lib/python3.10/site-packages/jinja2/environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "/nut/venv/lib/python3.10/site-packages/jinja2/defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "/nut/venv/lib/python3.10/site-packages/jinja2/filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/nut/venv/lib/python3.10/site-packages/markupsafe/__init__.py)

```

**Comment by Memodellocos** [comment]
installing markunsafe 2.0.1 worked for me as well.
 
<img width="634" alt="Screen Shot 2022-11-14 at 1 15 15 AM" src="https://user-images.githubusercontent.com/4770455/201700820-8a1af049-9a92-4a6c-b823-06c1b93e50f5.png">

**Comment by shawly** [comment]
Sorry for the late reply, the markupsafe issue was resolved in v3.3.2 but reintroduced with v3.3.3 so I didn't get why people still have this issue.

Since I was using the `edge` image I also didn't have this bug since the latest `master` branch from NUT doesn't have the markupsafe issue.

**Edit:** v3.3.4 should fix this issue again
